### PR TITLE
Implement `measures_actors` table and `measure_actors` API

### PR DESCRIPTION
**Pull request by parndt**
Implements #21  

We need a many to many join table describing a "passive" link between an
actor, introduced for #18, and a `measure` (actors are understood as
being targets or beneficiary of an activity, or 'measure'), from impact
(and updated in #31).

##### Permissions by role

- public: none
- registered, no role: none
- analyst: R
- manager: CRUD
- admin: CRUD

##### Rules

- only passive/target actors where allowed for its type (`actor.actortype.is_target = true`)
- only measures where allowed for its type (`measure.measuretype.has_target = true`)

### CODE REVIEW

**Comment by parndt** [comment]
This has code from #56 and #61 and those will need to be merged first.

**Comment by parndt** [comment]
> This has code from #56 and #61 and those will need to be merged first.

Now just relies on #56

File: app/models/actor_measure.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class ActorMeasure < VersionedRecord
+  self.table_name = :actors_measures
```
Comment (tmfrnz) [inline app/models/actor_measure.rb]: should be called actor_measures

File: app/models/measure_actor.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class MeasureActor < VersionedRecord
+  self.table_name = :measures_actors
```
Comment (tmfrnz) [inline app/models/measure_actor.rb:2]: table name should be measure_actors

File: db/schema.rb [hunk 3]
```diff
@@ -212,6 +212,22 @@
     t.index ["parent_id"], name: "index_measures_on_parent_id"
   end
 
+  create_table "measures_actors", force: :cascade do |t|
```
Comment (tmfrnz) [inline db/schema.rb:215]: dito

**Comment by parndt** [comment]
> Now just relies on #56

✅

**Review by tmfrnz — APPROVED** [review]
again approving, while noting naming should be made consistent eventually

**Pull request merged**